src/main/java/org/mskcc/cbio/oncokb/service/TokenService.java

```diff
@@ -58,11 +58,13 @@ public interface TokenService {
 
     void expireToken(Token token);
 
+    void delete(Long id);
```

Comment:
The original delete works with the following assumptions:

```
Ideally, users should have at most two tokens, so deleting one will just mean that
we assign the token's expiration to the other token.
In case where user has more than two tokens, we apply the expiration of the longest token
to the second longest token.
```

I thought it would be simplest to create a delete without side effects than try to modify the current implementation and potentially break existing features.